app/controllers/all-service-transactions/get.controller.js

```diff
@@ -20,26 +20,34 @@ const { CORRELATION_HEADER } = require('../../utils/correlation-header.js')
 module.exports = async function getTransactionsForAllServices (req, res, next) {
   const correlationId = req.headers[CORRELATION_HEADER] || ''
   const filters = getFilters(req)
+
+  // a filter param will be set on status specific routes, if they're not set the
+  // default behaviour should be live
+  const { statusFilter } = req.params
+  const filterLiveAccounts = statusFilter !== 'test'
+
   try {
-    const userPermittedAccountsSummary = await permissions.getLiveGatewayAccountsFor(req.user, 'transactions:read')
+    const userPermittedAccountsSummary = await permissions.getGatewayAccountsFor(req.user, filterLiveAccounts, 'transactions:read')
 
     const logContext = {
       gateway_account_ids: userPermittedAccountsSummary.gatewayAccountIds,
       user_number_of_live_services: req.user.numberOfLiveServices,
-      internal_user: req.user.internalUser
+      internal_user: req.user.internalUser,
+      is_live: filterLiveAccounts
     }
     logContext[keys.USER_EXTERNAL_ID] = req.user && req.user.externalId
     logContext[keys.CORRELATION_ID] = correlationId
-    logger.info('Listing all live services transactions', logContext)
+    logger.info('Listing all services transactions', logContext)
 
     if (!userPermittedAccountsSummary.gatewayAccountIds.length) {
-      return next(new NoServicesWithPermissionError('You do not have any associated services with rights to view live transactions.'))
+      return next(new NoServicesWithPermissionError('You do not have any associated services with rights to view these transactions.'))
     }
     const searchResultOutput = await transactionService.search(userPermittedAccountsSummary.gatewayAccountIds, filters.result)
     const cardTypes = await client.getAllCardTypes(correlationId)
-    const model = buildPaymentList(searchResultOutput, cardTypes, null, filters.result, router.paths.allServiceTransactions.download, req.session.backPath)
+    const route = filterLiveAccounts ? paths.allServiceTransactions.download : paths.formattedPathFor(paths.allServiceTransactions.downloadStatusFilter, 'test')
```

Comment:
(minor) can we say `downloadRoute` to make it clear what it is for ?

Reply:
Yeah good suggestion I think, it was called route as that's the param name for the code that then uses it but this should help clarify in context